solarkraft/package.json

```diff
@@ -30,14 +30,15 @@
     "dist/**/*"
   ],
   "scripts": {
+    "alert": "mocha --timeout 0 --loader=ts-node/esm test/alert/**/*.test.ts",
```

Comment:
Why not put it under `e2e` which already exists?

Reply:
Well, partially because it's not really end-to-end, more like mid-to-end, but more prominently, because this way it gets a separate timeout flag, since it's currently doing some back-and-forth requests each time, and so the test takes longer than all the rest (which have a default of 2s timeout)

Reply:
You can set the timeout. See the e2e tests